stixcore/idb/manager.py

```diff
@@ -1,13 +1,28 @@
 from datetime import datetime
```

Comment:
imports are usually grouped into three sections with import x before from x import y in each section

python standard lib
third party
localpacage

isort should be automatically running and doing this but it seems not to be

Reply:
Ok I need to set this up and one or two other things

Reply:
so we will fix it later automatically